torch/fx/experimental/symbolic_shapes.py

```diff
@@ -705,6 +715,12 @@ def safe_expand(r):
 def error():
     raise AssertionError("shouldn't be hit")
 
+
+def get_debugging_stack():
+    # cut this frame and the caller's frame
+    return ''.join(traceback.format_list(traceback.extract_stack()[:-2]))
```

Comment:
nit: I find it helpful to take in the number of frames to remove as an argument to this function

Reply:
done